3DAmsterdam/Assets/Netherlands3D/Scripts/Utilities/ConvertCoordinates.cs

```diff
@@ -45,102 +63,52 @@ public Vector3WGS(double Lon, double Lat, double H)
-        //referenceRD can be used to check de desired origin, and to set referenceWGS84
-        public static Vector3WGS referenceWGS84 = new Vector3WGS(4.892504, 52.373043,0);
-        public static Vector3RD referenceRD = new Vector3RD(121311.223, 487355.782,-42.962);
-
         //scalefactors for converting WGS84 to Unity
         private static double unitsPerDegreeX = 67800;  //approximation of distance between longitudinal degrees in meters at reference-lattitude
         private static double unitsPerDegreeY = 111000; //approximation of distance between lattitudinal degrees in meters
 
-
         private static byte[] RDCorrectionX = Resources.Load<TextAsset>("x2c").bytes;
         private static byte[] RDCorrectionY = Resources.Load<TextAsset>("y2c").bytes;
         private static byte[] RDCorrectionZ = Resources.Load<TextAsset>("nlgeo04").bytes;
 
         private static Vector3RD output = new Vector3RD();
+
+        private static Vector3WGS? _referenceWGS84  = null;
+
         /// <summary>
-        /// set ReferenceWGS84 to given value, 
-        /// sets referenceRD to corresponding value, 
-        /// sets unitsperdegreeX based on unitsperdegreeY and referenceWGS.y
+        /// set ReferenceWGS84 converted from RelativeCenterRD,
         /// </summary>
         public static Vector3WGS ReferenceWGS84
         {
             get
             {
-                return referenceWGS84;
-            }
-            set
-            {
-                //set value
-                referenceWGS84 = value;
-                // update unitsPerDegreeX
-                unitsPerDegreeX = Math.Cos(referenceWGS84.lat * Math.PI / 180) * unitsPerDegreeY;
-                // update referenceRD
-                Vector3RD refRD = WGS84toRD(referenceWGS84.lon, referenceWGS84.lat);
-                // get elevation 
-                double hoogte = RDCorrection(referenceWGS84.lon,referenceWGS84.lat,"Z",RDCorrectionZ);
-                referenceRD = new Vector3RD(refRD.x,refRD.y,-hoogte);
-                
-            }
-        }
-       
-        public static Vector3RD ReferenceRD
-        {
-            get
-            {
-                return referenceRD;
-            }
-            set
-            {
-                //set value
-                referenceRD = value;
-                //update referenceWGS84
-                Vector3WGS refWGS = RDtoWGS84(referenceRD.x, referenceRD.y);
-                double hoogte = RDCorrection(refWGS.lon, refWGS.lat, "Z", RDCorrectionZ);
-                referenceWGS84 = new Vector3WGS(refWGS.lon, refWGS.lat,hoogte);
-                //update unitsperdegreeX
-                unitsPerDegreeX = Math.Cos(referenceWGS84.lon * Math.PI / 180) * unitsPerDegreeY;
+                if(_referenceWGS84 == null)
+                {
+                    _referenceWGS84 = RDtoWGS84(Config.activeConfiguration.RelativeCenterRD.x, Config.activeConfiguration.RelativeCenterRD.y);                    
+                }
```

Comment:
should also calculate the wgs-height (like in old lines 102-103). wgs-hegiht is not rd-height

Reply:
Thanks, I made an update